lte/gateway/python/magma/enodebd/tr069/models.py

```diff
@@ -308,6 +308,19 @@ class Reboot(Tr069ComplexModel):
     _type_info["CommandKey"] = CommandKeyType
 
 
+class FactoryReset(Tr069ComplexModel):
+    """
+    no input parameters
+    """
```

Comment:
nit - put this in a single line
```suggestion
    """ no input parameters """
```

Reply:
nit - The first content in the docstring should describe what the class is

Reply:
Modify.